### DISCUSSION TITLE
How do I generate Shell_NotifyIcon?

### DISCUSSION THREAD

**Discussion by nathanAjacobs** [microsoft/CsWin32#1155] [Q&A] [answered]
I've added Shell_NotifyIcon to the NativeMethods.txt file, but it is not generating.

**Comment by AArnott** [comment, marked as answer]
Did you try building or looking at the error list? This warning might be produced:

> warning PInvoke005: This API is only available when targeting a specific CPU architecture. AnyCPU cannot generate this API.

And that would explain why you don't see it.

**Reply by nathanAjacobs** [reply]
Thank you for your response! I thought I had my project setup to target a specific architecture, but I indeed did not. That fixed it :)
